docs/governance/fees/fees.md

```diff
@@ -35,9 +35,12 @@ The purpose of this page is to let users know which fee models are active at any
 
 ### Volume fee on Gnosis Chain orders (excluding stable-to-stable pairs)
 
+[//]: # (If updating the list of stable coins, do not forget to update the actual code where this is handled on CoW Swap)
```

Comment:
I was meaning more the other way around. To add a message in the code pointing to this markdown file, so anyone changing the list in the code would also come here.

Alternativelly, we can make our life easier but not adding the detailed list (just some examples), and then mention the complete list is specified in code (and the you link the lines as you did here)

Reply:
I did that too https://github.com/cowprotocol/cowswap/pull/4647